x/concentrated-liquidity/lp_test.go

```diff
@@ -1121,19 +1124,19 @@ func (s *KeeperTestSuite) TestInverseRelation_CreatePosition_WithdrawPosition()
 			// 1. amount for denom0 and denom1 upon creating and withdraw position should be same
 			// Note: subtracting one because create position rounds in favor of the pool.
 			s.Require().Equal(amtDenom0CreatePosition.Sub(sdk.OneInt()).String(), amtDenom0WithdrawPosition.String())
-			s.Require().Equal(amtDenom1CreatePosition.String(), amtDenom1WithdrawPosition.String())
+			s.Require().Equal(amtDenom1CreatePosition.Sub(sdk.OneInt()).String(), amtDenom1WithdrawPosition.String())
 
 			// 2. user balance and pool balance after creating / withdrawing position should be same
 			userBalancePostPositionCreation := s.App.BankKeeper.GetAllBalances(s.Ctx, s.TestAccs[0])
 			poolBalancePostPositionCreation := s.App.BankKeeper.GetAllBalances(s.Ctx, poolBefore.GetAddress())
 
-			// Note: subtracing one since position creation rounds in favor of the pool.
+			// Note: subtracting one since position creation rounds in favor of the pool.
 			s.Require().Equal(userBalancePrePositionCreation.AmountOf(ETH).Sub(sdk.OneInt()).String(), userBalancePostPositionCreation.AmountOf(ETH).String())
-			s.Require().Equal(userBalancePrePositionCreation.AmountOf(USDC).String(), userBalancePostPositionCreation.AmountOf(USDC).String())
+			s.Require().Equal(userBalancePrePositionCreation.AmountOf(USDC).Sub(sdk.OneInt()).String(), userBalancePostPositionCreation.AmountOf(USDC).String())
 
 			// Note: adding one since withdrawal rounds in favor of the pool.
 			s.Require().Equal(poolBalancePrePositionCreation.AmountOf(ETH).Add(sdk.OneInt()).String(), poolBalancePostPositionCreation.AmountOf(ETH).String())
-			s.Require().Equal(poolBalancePrePositionCreation.AmountOf(USDC).String(), poolBalancePostPositionCreation.AmountOf(USDC).String())
+			s.Require().Equal(poolBalancePrePositionCreation.AmountOf(USDC).Add(sdk.OneInt()).String(), poolBalancePostPositionCreation.AmountOf(USDC).String())
```

Comment:
Thoughts on having a variable called `roundingError` that's set to `OneInt()` for test readability?

Reply:
done